realm/realm-library/src/main/java/io/realm/internal/async/BadVersionException.java

```diff
@@ -24,7 +23,7 @@
  * state of the Realm which might be more up-to-date than the provided results or vice versa.
  */
 @Keep
-public class BadVersionException extends RealmException {
+public class BadVersionException extends Exception {
 
     public BadVersionException(String detailMessage) {
```

Comment:
Sure - create another issues for that. As long as it's not exposed to the user API.